gto-support-db/src/main/java/org/ccci/gto/android/common/db/AbstractDao.java

```diff
@@ -216,9 +216,10 @@ public final Cursor getCursor(@NonNull final Class<?> clazz, @NonNull final Stri
     @WorkerThread
     public final <T> Cursor getCursor(@NonNull final Class<T> clazz, @NonNull final Join<T, ?>[] joins,
                                       @NonNull final String[] projection, @Nullable final String whereClause,
-                                      @Nullable final String[] whereArgs, @Nullable String orderBy) {
+                                      @Nullable final String[] whereArgs, @Nullable String orderBy,
+                                      @Nullable final String groupBy) {
```

Comment:
let's not add groupBy to this method, it's deprecated and kept around for compatibility with projects I haven't converted to use the Query object yet.